app/server_inactivity.go

```diff
@@ -27,58 +32,22 @@ func (s *Server) doInactivityCheck() {
 		return
 	}
 
+	_, sysValErr := s.Store.System().GetByName(INACTIVITY_EMAIL_SENT)
```

Comment:
Do we need to keep checking the `INACTIVITY` entry to not resend to installations who have already seen the email once?